pkg/daemon/writer.go

```diff
@@ -29,14 +29,16 @@ type NodeStateStatusWriter struct {
 	OnHeartbeatFailure func()
 	metaData           *utils.OSPMetaData
 	networkData        *utils.OSPNetworkData
+	devMode            bool
 }
 
 // NewNodeStateStatusWriter Create a new NodeStateStatusWriter
-func NewNodeStateStatusWriter(c snclientset.Interface, n string, f func()) *NodeStateStatusWriter {
+func NewNodeStateStatusWriter(c snclientset.Interface, n string, f func(), devMode bool) *NodeStateStatusWriter {
 	return &NodeStateStatusWriter{
 		client:             c,
 		node:               n,
 		OnHeartbeatFailure: f,
+		devMode:            devMode,
```

Comment:
Since this is later passed to `utils.DiscoverSriovDevices`, I'm wondering if it makes any sense to keep the name "devMode" here. Wouldn't it make more sense to also rename to `withUnsupported` or even `withUnsupportedDevices`?

Reply:
Sounds good. Fixed.